client/lib/inflight/test/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import inflight from '../';
+
+const randomRequestKey = () => `${ Math.random() }`;
+const THE_ANSWER = 42;
+
+describe( 'inflight', ( ) => {
+	describe( 'promiseTracker', () => {
+		it( 'should mark an item as inflight while the request is ongoing', ( done ) => {
+			const key = randomRequestKey();
```

Comment:
removed randomness in https://github.com/Automattic/wp-calypso/pull/11069/commits/a6ea2c961b1f0c1aa6b4d4f0d59f79b209cb7307